tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -222,19 +240,26 @@ <h3>No profile data was found.</h3>
       if (oldDatasets != null || this.selected == null) {
         // Select the first dataset by default.
         this.set('selectedDataset', 0);
+        this.set('currentSelected', 0);
       }
     },
     _selectedDatasetChanged: function(newDataset, oldDataset) {
       if (this._datasets) {
         // Display the first tool by default when switching to another run.
-        this.set('selectedTool', 0);
+        this.set('currentSelected', (newDataset << 16));
+
+        // Same tool can have differernt index in different runs, therefore we force a change of
+        // 'selectedTool', to make sure everything is synced.
```

Comment:
Correct me if applicable, but it seems like what we want to achieve here is to run the logic within `_selectedToolChanged`. 

Could we refactor the logic within `_selectedToolChanged` into a new `_computeCurrentTool` method? And then call `_computeCurrentTool` right here instead of setting `selectedTool` to undefined then 0?

Reply:
Also, we don't need bind if we use an arrow method, ie `() => { ... }`.

Reply:
b/71633342 #4 explain why we can not use selectedToolchanged.